src/js/deta-modals.js

```diff
@@ -28,8 +28,11 @@ const detaModals = {
                     } else if (res.status === 200) {
                         b2.blur();
                         successHandler(filename);
+                    } else if (res.status === 401) {
+                        document.getElementById("save_warning").innerHTML = `There was an error while saving the drawing. Please refresh the page, and download any work to prevent data loss.`
```

Comment:
`There was an authentication issue while saving the drawing. Please download any unsaved work ('export svg'), refresh the page, and try again.`